dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/merge/dataelement/DataElementMergeServiceTest.java

```diff
@@ -2546,8 +2555,9 @@ void dataValueAuditMergeDeleteTest() throws ConflictException {
   // --------------------------------------
   @Test
   @DisplayName(
-      "TrackedEntityDataValueChangeLog with references to source DataElements are not changed or deleted when sources not deleted")
-  void trackedEntityDataValueChangeLogMergeTest() throws ConflictException {
+      "TrackedEntityChangeLogs with references to source DataElements are not changed or deleted when sources not deleted")
+  void trackedEntityChangeLogMergeTest()
```

Comment:
You renamed the test but kept the tracked entity in here even though the test seems to test `EventChangeLog...` things. Should the test not be named after `EventChangeLog` somehow? 😅